content/en/getting-started/installation.md

```diff
@@ -1,155 +1,206 @@
 ---
 title: "Installation"
 linkTitle: "Installation"
-weight: 3
+weight: 10
 description: >
   Basic installation guide to get started with LocalStack on your local machine.
 cascade:
   type: docs
 ---
```

Comment:
<img width="729" alt="Screenshot 2023-06-22 at 18 33 02" src="https://github.com/localstack/docs/assets/8161292/e10b4346-af54-4c3c-b8eb-9567d85988e9">

The 10 min could appear deterring for a seemingly simple and quick installation. We could consider hiding it (`hide_readingtime: true`), bringing it at least below two digits, or moving all alternatives to another page (might break existing anchor links).